### ISSUE TITLE
Something went wrong #BRL4

### ISSUE THREAD

**Issue by tongyi00** [Jaammerr/The-Dawn-Bot#31]
"Something went wrong #BRL4". Does this error mean that the account was banned because it was found to be use script?

**Comment by Jaammerr** [comment]
Nope, this means that your account was banned because DAVN restricted the use of this domain
